Soccar [Game-Server]/src/nl/soccar/gameserver/util/CarUtilities.java

```diff
@@ -0,0 +1,69 @@
+package nl.soccar.gameserver.util;
+
+import javafx.scene.shape.Rectangle;
+import nl.soccar.library.*;
+import nl.soccar.library.enumeration.TeamColour;
+import nl.soccar.physics.GameEngine;
+import nl.soccar.physics.models.CarPhysics;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * @author PTS34A
+ */
+public final class CarUtilities {
+
+    private CarUtilities() {
+    }
+
+    public static void addCars(GameEngine engine, Map map, Team blue, Team red) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)